analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    dir = 'C:/Users/AdithiPriya/Desktop/Hack for LA/Geospatial analysis'
</pre><p>Can we make this point to the current directory, so that other data scientists could clone this repo and run the code directly? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>